### PR TITLE
Implementation of flavor feature

### PR DESCRIPTION
**Pull request by DblK**
The flavor feature allow you to select the flavor "normal", "sdk" when
using nw-builder.

### CODE REVIEW

**Comment by DblK** [comment]
It's normal everything failed because I changed the signature of the functions.
It needs to adjust test and also write new tests.

File: lib/Version.js [hunk 1]
```diff
@@ -32,24 +34,26 @@ module.exports = function Version(args){
             return platform.replace('32', '-ia32').replace('64', '-x64');
         };
 
-        var mapPlatformToExtension = function(platform){
+        var mapPlatformToExtension = function(platform, flavor){
```
Comment (adam-lynch) [inline lib/Version.js]: This isn't used?

**Comment by adam-lynch** [comment]
Thanks for the pull-request! I published the fix for supporting newer versions today and had to make a couple of fixes. Sorry, do you mind basing this of master instead? 😊

**Comment by DblK** [comment]
Okay, I will try to do it soon enough.

**Comment by DblK** [comment]
See #340

**Pull request closed without merging**